### DISCUSSION TITLE
Setting the style with 3.3.11

### DISCUSSION THREAD

**Discussion by charesch** [HandyOrg/HandyControl#1118] [Q&A] [answered]
I used to be able to set a style using Style="{StaticResource TextBoxExtend}" with an older version. 

How would I explore available styles with the current version?

Thank you very much for your time and effort!

**Comment by ghost1372** [comment]
Check our [docs](https://hosseini.ninja/) you can find all available styles or you can explore source code

**Comment by charesch** [comment]
Great! Got it almost working. How do I set a <hc:WatermarkTextBox /> with the TextBoxExtend Style? When i set the style it removes the watermark and when i dont set it the style seems empty.

**Reply by ghost1372** [reply]
Dont use watermark, use TextBox and TextBoxExtend style with InfoElement.PlaceHolder... attribute

**Comment by charesch** [comment, marked as answer]
Thank you!  <hc:TextBox hc:InfoElement.Placeholder="Watermark" /> was the answer!
